bundler/spec/dependabot/bundler/file_updater_spec.rb

```diff
@@ -439,6 +439,27 @@
         end
       end
 
+      context "with an imported gemspec that specifies a minimum Ruby version not satisfied by the running Ruby" do
+        let(:dependency_files) { bundler_project_dependency_files("unsatisfied_required_ruby_version") }
+
+        before do
+          require "dependabot/bundler/file_updater/ruby_requirement_setter"
+
+          stub_const(
+            "#{described_class}::RubyRequirementSetter::RUBY_VERSIONS",
+            described_class::RubyRequirementSetter::RUBY_VERSIONS + ["99.0.0"]
+          )
```

Comment:
I could've used `>= 3.1` in the fixture gemspec, and that would've worked, but once Dependabot upgrades to Ruby 3.1, the test would stop covering this fix. So this is my attempt to make the test independent from the Ruby version being run.